std_msgs/Header header

geometry_msgs/Vector3 cur_pos
geometry_msgs/Quaternion cur_ori
geometry_msgs/Vector3 cur_vel
geometry_msgs/Vector3 cur_vel_body
geometry_msgs/Vector3 cur_Omega
geometry_msgs/Vector3 cur_Omega_body

std_msgs/Float32 yaw
std_msgs/Float32 pitch
std_msgs/Float32 roll
std_msgs/Float32 yaw_d
std_msgs/Float32 pitch_d
std_msgs/Float32 roll_d

std_msgs/Float32 dot_yaw

std_msgs/Float32 theta_e

std_msgs/Float32 angular_velocity
std_msgs/Float32 linear_velocity

std_msgs/Float32 thrust_cmd
geometry_msgs/Quaternion q_cmd

std_msgs/Float32 T_x
std_msgs/Float32 T_y

float64 drag
float64 accd
